iceoryx_hoofs/test/moduletests/test_resizeable_lockfree_queue.cpp

```diff
@@ -481,7 +471,8 @@ TYPED_TEST(ResizeableLockFreeQueueTest, DecreaseCapacityOfAPartiallyFilledQueue)
 
     uint64_t element = 0U;
     while (q.tryPush(element++))
-        ;
+    {
+    }
```

Comment:
We should have a clang-format which creates `while (cond) {}` in a single line

Reply:
Or, better, allow the previous code which is not bad to read and it is perfectly fine to have a loop and some small amount of logic in the loop condition. Textbook example would be quicksort (splitting by pivot).

Reply:
No, not in this but maybe in a later PR.